mix.exs

```diff
@@ -40,16 +40,13 @@ defmodule ElixirBoilerplate.Mixfile do
 
   defp deps do
     [
-      # HTTP Client
-      {:hackney, "~> 1.16"},
```

Comment:
Why do we remove this deps?

Reply:
I thought we did not have to explicitly define de dependency, but it looks like `new_relic`needs an explicit definition…